src/functionalTest/resources/application-integration.yml

```diff
@@ -123,3 +123,21 @@ mail:
 
 idempotent-flag-ingestion: true
 fetch-personal-elinks-id: select elinks_id from judicial_user_profile
+
+insert-job-sql: insert into dataload_schedular_job(job_start_time, publishing_status) values(?, ?)
+update-job-sql: update dataload_schedular_job set job_end_time = now() at time zone 'utc', publishing_status = ? where id =?
+select-job-status-sql:  select publishing_status from dataload_schedular_job where date(job_start_time) = current_date
+get-sidam-ids: select sidam_id from judicial_user_profile where sidam_id is not null
+
+jrd:
+  publisher:
+    azure:
+      service:
+        bus:
+          host: ${JRD_MQ_HOST:rd-servicebus-sandbox.servicebus.windows.net}
+          username: ${JRD_MQ_USERNAME:SendAndListenSharedAccessKey}
+          password: ${JRD_MQ_PASSWORD:AAAAAAAAAA}
+          topic: ${JRD_MQ_TOPIC_NAME:rd-caseworker-topic-sandbox}
```

Comment:
should this be topic related to JRD instead of CRD

Reply:
done